src/lib/components/buttons/file-chooser/terra-file-chooser.component.ts

```diff
@@ -25,11 +24,7 @@ export class TerraFileChooserComponent extends TerraButtonComponent {
     }
 
     public get inputPrimaryBrowserButtonCaption(): string {
-        if (!isNullOrUndefined(this._primaryBrowserButtonCaption) && this._primaryBrowserButtonCaption.length > 0) {
-            return this._primaryBrowserButtonCaption;
-        }
-
-        return this._translation.translate(this._translationPrefix + '.choose');
+        return this._primaryBrowserButtonCaption || this._translation.translate(this._translationPrefix + '.choose');
```

Comment:
&& his._primaryBrowserButtonCaption.length > 0. War es zumindest zuvor

Reply:
It's the same, just. The caption is a string, so '' would be falsy.